### PR TITLE
Fix UI issues and add language switcher to imprint

### PR DESCRIPTION
**Pull request by ulliholtgrave**
### Short description

This PR fixes the UI of the versioning view, makes some small changes to the page form and adds the language switcher to the imprint form.

### Proposed changes

- Change CSS of versioning slider
- Add language switcher to imprint form and add check if content has no id
- Add some small UI tweaks to the language switcher

### Resolved issues

Fixes: #1112 #1111

### CODE REVIEW

File: integreat_cms/cms/templates/content_form_language_switcher.html [hunk 1]
```diff
@@ -4,12 +4,12 @@
 {% load i18n %}
 
 <div id="language-switcher" class="flex flex-col z-10 text-lg">
-    <span id="language-switcher-current" href="{% build_url target=target region_slug=request.region.slug language_slug=language.slug content_field=content_field content_id=content_id %}" class="font-bold flex justify-between bg-white py-2 px-4 rounded shadow hover:bg-gray-200 border-2 border-white hover:border-gray-200">
-        Weitere Sprachen
+    <span id="language-switcher-current" class="font-bold flex justify-between bg-white py-2 px-4 rounded shadow hover:bg-gray-200 border-2 border-white hover:border-gray-200">
```
Comment (timoludwig) [inline integreat_cms/cms/templates/content_form_language_switcher.html:7]:
```suggestion
    <span id="language-switcher-current" class="font-bold flex justify-between bg-white py-2 px-4 rounded shadow border-2 border-white">
```
I'm not sure about this, but if this area itself isn't a link anymore and clicking on it does nothing, maybe we should remove the hover effects and the pointer? But I don't know what's best practice for UI elements which trigger an action on _hovering_, not on _clicking_...

File: integreat_cms/cms/templates/content_form_language_switcher.html [hunk 2]
```diff
@@ -4,12 +4,12 @@
...
+        {% trans 'More Languages' %}
     </span>
     <div id="language-switcher-list" class="h-0 hidden">
         {% for other_language in languages|sort_languages:language|slice:"4:" %}
-            <a href="{% build_url target=target region_slug=request.region.slug language_slug=other_language.slug content_field=content_field content_id=content_id %}" class="block flex justify-between whitespace-nowrap {% if forloop.last %} rounded-b {% endif %} shadow px-4 py-3 text-gray-800 hover:bg-gray-200 bg-white">
+            <a href="{% if target != 'edit_imprint' %}{% build_url target=target region_slug=request.region.slug language_slug=other_language.slug content_field=content_field content_id=content_id %}{% else %} {% url 'edit_imprint' region_slug=request.region.slug language_slug=other_language.slug %} {% endif %}" class="block flex justify-between whitespace-nowrap {% if forloop.last %} rounded-b {% endif %} shadow px-4 py-3 text-gray-800 hover:bg-gray-200 bg-white">
```
Comment (timoludwig) [inline integreat_cms/cms/templates/content_form_language_switcher.html:12]: I guess this condition could also be handled in the `build_url` template tag by passing the `content_field` to the reverse function only when it's not `None`...

**Review by timoludwig — APPROVED** [review]
Thanks a lot, looks good to me! :+1:
Just some minor details, but they are optional I think.

**Comment by ulliholtgrave** [comment]
All changes are included in #1116

**Pull request closed without merging**